src/ucp/core/ucp_worker.c

```diff
@@ -2193,7 +2193,7 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
      */
     ucs_config_parser_warn_unused_env_vars_once(context->config.env_prefix);
 
-    ucs_vfs_obj_add_dir(context, worker, "worker/%p", worker);
+    ucs_vfs_obj_add_dir(context, worker, "worker/%s-%p", worker->name, worker);
```

Comment:
imo, better use just worker name, otherwise, with default worker name, its pointer will be printed twice (note default worker name is a pointer)

Reply:
This approach is consistent with UCP context https://github.com/openucx/ucx/blob/95b7c31ef866e4924391fe007978c93cbb297109/src/ucp/core/ucp_context.c#L1558

There was a discussion in similar PR for UCP context 
https://github.com/openucx/ucx/pull/6505#discussion_r595822904

However, I agree with you now. The ui of VFS tree will be a little better IMO.

Reply:
We have this:
![1](https://user-images.githubusercontent.com/74596089/113607065-75ee4900-9651-11eb-9222-f5356d684bb8.png)

How about that?
![2](https://user-images.githubusercontent.com/74596089/113607146-8b637300-9651-11eb-9838-c17719670325.png)

Reply:
right, missed that part. I thought we agreed to show pointer as a separate RO object.
maybe better do this?

Reply:
let's do it with name only, both for context and worker

Reply:
And I think we should do the same for context.

Reply:
> let's do it with name only, both for context and worker

Ok